sdk/src/asset_io.rs

```diff
@@ -53,6 +58,16 @@ pub trait CAILoader {
     fn read_xmp(&self, asset_reader: &mut dyn CAIRead) -> Option<String>;
 }
 
+pub trait CAIWriter {
+    // Return entire CAI block as Vec<u8>
```

Comment:
"Return entire block?" … but the return type is `Result<()>`

Reply:
yeah that changed, thanks for catching it.